core/sets.gms

```diff
@@ -2502,7 +2509,41 @@ se2fe(all_enty,all_enty,all_te)   "map secondary energy to end-use energy using
         seliqsyn.fedie.tdsyndie
         seh2.feh2t.tdh2t
 /
+
+
 sefe(all_enty,all_enty) "map secondary energy to final energy"
+//
+
+entySe2entyFe(all_enty,all_enty) "map secondary energy to final energy"
```

Comment:
I feel we should have had such a mapping already, no?

Reply:
you shoudl use the set `sefe` instead.